SourceX/storm/storm_net.cpp

```diff
@@ -111,12 +126,20 @@ BOOL SNetCreateGame(const char *pszGameName, const char *pszGamePassword, const
 
 	char addrstr[129] = "0.0.0.0";
 	SRegLoadString("dvlnet", "bindaddr", 0, addrstr, 128);
+	strncpy(gpszGameName, addrstr, sizeof(gpszGameName) - 1);
+	if (pszGamePassword)
+		strncpy(gpszGamePassword, pszGamePassword, sizeof(gpszGamePassword) - 1);
 	*playerID = dvlnet_inst->create(addrstr, pszGamePassword);
 	return *playerID != -1;
 }
 
-BOOL SNetJoinGame(int id, char *pszGameName, char *pszGamePassword, char *playerName, char *userStats, int *playerID)
+BOOL SNetJoinGame(int id, char *pszGameName, char *pszGamePassword, char *playerName,

```

Comment:
why break in 2 lines? :P other than that, looks good to me

Reply:
I've run clang-format -i SourceX/storm/storm_net.cpp -style=file on the file to be sure to match the project format.
I was thinking that this modification came from that, but, in fact, it's from my own tools and it's just fine for clang-format
Anyway, I'll remove it :)